prometheus/Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN \
     && pip3 install \
         --no-cache-dir \
         --prefer-binary \
-        --find-links "https://wheels.home-assistant.io/alpine-3.15/${BUILD_ARCH}/" \
+        --find-links "https://wheels.home-assistant.io/alpine-3.14/${BUILD_ARCH}/" \
```

Comment:
This is incorrect, as a matter of fact, the image is not 3.14, please remove this change.

Reply:
https://wheels.home-assistant.io/alpine-3.15/amd64/ gives a 404 error, whereas alpine-3.14 works.
![image](https://user-images.githubusercontent.com/35043496/180101220-b527cb2d-ae94-449b-90f4-ab096b570e80.png)

Reply:
Correct :)
That is fine for now.

This code will fully change in the new base images I'm rolling out anyways

Reply:
Are you saying that is fine to leave it as 3.14 for now, or that it is fine for the find-links to fail when the version is 3.15?